packages/web-runtime/src/pages/account.vue

```diff
@@ -73,21 +73,30 @@
 
 <script>
 import { mapGetters } from 'vuex'
+import { useCapabilityFilesSharingResharing } from '../composables'
 export default {
   name: 'Personal',
+  setup() {
+    return {
+      // FIXME: We need a clean way to know whether account editing is enabled or not
+      // for now we abuse the files_sharing.resharing capability, as *currently*
+      // oCIS does not support that exactly like it does not support account editing
+      isAccountEditingEnabled: useCapabilityFilesSharingResharing()
```

Comment:
Please check instead if the `apps` store module from the runtime has the `settings` app. That's a more reliable indicator for now whether we run ocis or not.